logging/interfaces.go

```diff
@@ -0,0 +1,12 @@
+package logging
+
+// EventBuilder interface is an abstraction over logger.EventBuilder that allows to build log events with custom tags
```

Comment:
```suggestion
// EventBuilder allows to build log events with custom tags.
```

I think now `abstraction over ...` is redundant, because abstraction and implementation in one pkg (near to each other)

Reply:
Or fix it to `abstraction over zerolog_adapter.EventBuilder`

Reply:
Changed